backend/src/users/users.service.ts

```diff
@@ -24,9 +24,12 @@ export class UsersService {
       throw new ConflictException('Email d\'utilisateur déjà pris');
     }
 
-    const defaultRole = await this.roleModel.findOne({ name: 'employee' });
+    let defaultRole = await this.roleModel.findOne({ name: 'employee' });
     if (!defaultRole) {
-      throw new NotFoundException('Le rôle "employee" par défaut est introuvable');
+      defaultRole = await this.roleModel.create({
```

Comment:
When the database is created roles are already set up. No need to remove the exception ?